### ISSUE TITLE
how to get kusama address from mnemonic

### ISSUE THREAD

**Issue by soultrain23** [polkascan/py-substrate-interface#153]
I can't get the kusama address from mnemonic. Only ss58 addresses come out.

I can't find an explanation for what processing the address I use as the account address can be made.

**Comment by arjanz** [comment]
I think there is some confusing about what the SS58 format is. 

It is a certain way to encode the public key so it will contain information about the network it is intended to be used and a checksum to prevent typos. More information: https://docs.substrate.io/v3/advanced/ss58/

So every network has ID that will be used to encode the address, by default for development that is `42` and will look like: `5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY`

For Kusama that ID is `2` and for Polkadot `0`. 

To see what that ID is you can check:

`substrate.ss58_format` 

So basically you can use:

```python
substrate = SubstrateInterface(
    url=KUSAMA_WS_URL
)

mnemonic = Keypair.generate_mnemonic()
keypair = Keypair.create_from_mnemonic(mnemonic, ss58_format=substrate.ss58_format)
```

Hope this helps

**Comment by soultrain23** [comment]
Ive been confused something very long time. 
It's just 
keypair = Keypair.create_from_mnemonic(seed, ss58_format=2)
